addons/languages/english/pack/src/main/res/xml/qwerty.xml

```diff
@@ -52,7 +52,7 @@
         <Key android:codes="99" android:keyLabel="c" android:popupCharacters="çćĉčγ"/>
         <Key android:codes="118" android:keyLabel="v"/>
         <Key android:codes="98" android:keyLabel="b" android:popupCharacters="β"/>
-        <Key android:codes="110" android:keyLabel="n" android:popupCharacters="ñńν"/>
+        <Key android:codes="110" android:keyLabel="n" android:popupCharacters="ňñńν"/>
```

Comment:
@strnbrg59 does this order makes sense? Considering this keyboard is mainly for English speakers

Reply:
@JakubKubena I prefer `ň` be the last character in the list. Can you change that to `android:popupCharacters="ñńňν"`

Reply:
```suggestion
        <Key android:codes="110" android:keyLabel="n" android:popupCharacters="ñńňν"/>
```